ui/app/components/app/transaction-breakdown/transaction-breakdown.component.js

```diff
@@ -17,9 +17,9 @@ export default class TransactionBreakdown extends PureComponent {
     nativeCurrency: PropTypes.string,
     showFiat: PropTypes.bool,
     nonce: PropTypes.string,
+    primaryCurrency: PropTypes.string,
     gas: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
     gasPrice: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-    value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
```

Comment:
Nit: could you remove this from the container as well? It's still used in the container but it's no longer required to get passed down to this component.

Reply:
I did it in [3bb5bcf TxBreakdown Container L:24](https://github.com/MetaMask/metamask-extension/commit/3bb5bcff932c3d026b37903ece8d1ad58af35118#diff-79772a36e939c5b2a111313bc678686bL24).